configurer/path_windows.go

```diff
@@ -3,48 +3,84 @@
 package configurer
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
 	"syscall"
+
+	"github.com/pastelnetwork/pastel-utility/constants"
 )
 
 const (
 	beforeVistaAppDir = "Application Data"
 	sinceVistaAppDir  = "AppData/Roaming"
 )
 
+// GetHomeDir returns the home path for Windows OS.
+func GetHomeDir() (string, error) {
```

Comment:
what's the purpose of this function when you are using `os.UserHomeDir()` directly at many places in the same file. Either remove this wrapper function which is not serving any purpose or use this function in those places.